src/group_theory/group_action/prod.lean

```diff
@@ -41,6 +41,16 @@ variables [has_smul M α] [has_smul M β] [has_smul N α] [has_smul N β] (a : M
 @[to_additive] theorem smul_def (a : M) (x : α × β) : a • x = (a • x.1, a • x.2) := rfl
 @[simp, to_additive] theorem smul_swap : (a • x).swap = a • x.swap := rfl
 
+variables [has_pow α E] [has_pow β E] (c : E) (p : α × β)
+@[to_additive has_smul] instance has_pow : has_pow (α × β) E :=
+{ pow := λ p c, (p.1 ^ c, p.2 ^ c) }
+@[simp, to_additive smul_fst] lemma pow_fst : (p ^ c).fst = p.fst ^ c := rfl
+@[simp, to_additive smul_snd] lemma pow_snd : (p ^ c).snd = p.snd ^ c := rfl
+@[simp, to_additive smul_swap] lemma pow_swap : (p ^ c).swap = p.swap ^ c := rfl
+@[simp, to_additive smul_def] lemma pow_def : p ^ c = (p.1 ^ c, p.2 ^ c) := rfl
```

Comment:
```suggestion
@[to_additive smul_def] lemma pow_def : p ^ c = (p.1 ^ c, p.2 ^ c) := rfl
```
The smul version wasn't simp either

Can you put these in the same order as the smul lemmas too while you're here?